src/main/java/trello/entities/List.java

```diff
@@ -75,7 +75,7 @@ public void setId(final String id) {
      *
      * @param cards is the ArrayList that contains the cards to be set.
      */
-    public void setCards(final ArrayList<Card> cards) {
+    public void setCards(final java.util.List<Card> cards) {
```

Comment:
Please go up this import to the section of import.

Reply:
This is not an import. It is like that so the IDE couldn't confuse between the java.util.List and the entity List that we are using in the context.